src/forces.f90

```diff
@@ -506,12 +508,14 @@ subroutine force(ux1,uy1,ep1)
        !     endif
        if (nrank .eq. 0) then
           write(38,*) t,xDrag_mean,yLift_mean
-          call flush(38)
+          flush(38)
        endif
        if (mod(itime, icheckpoint).eq.0) then
           if (nrank .eq. 0) then
              write(filename,"('forces.dat',I7.7)") itime
-             call system("cp forces.dat " //filename)
+             !ierror=system("cp forces.dat " //filename)
+             call execute_command_line ("cp forces.dat "//filename, &
+                                        exitstat=ierror)
```

Comment:
Why do we need to collect the exit code of the command if nothing is done with it ?

Reply:
We can remove it, not a problem for me.